docs/development.md

```diff
@@ -217,7 +217,7 @@ https://onsi.github.io/ginkgo/#focused-specs
 ### Race conditions
 
 It is not uncommon that during the execution of the integration tests, test failures occur.
-Although it's possible that this is due to the tests themselves, more often than not this occurs due to the the actual odo code is written.
+Although it's possible that this is due to the tests themselves, more often than not this occurs due to the actual odo code is written.
```

Comment:
```...actual odo code...``` or ```...actual Odo code...``` ?

@kadel @cdrage Please confirm.
Otherwise looks good to me

Reply:
odo.  odo shouldn't be capitalized. That is the current standard that we should use.